litestar/config/app.py

```diff
@@ -207,16 +208,7 @@ class AppConfig:
     multipart_form_part_limit: int = field(default=1000)
     """The maximal number of allowed parts in a multipart/formdata request. This limit is intended to protect from
     DoS attacks."""
-    experimental_features: list[ExperimentalFeatures] | None = None
-
-    def __post_init__(self) -> None:
-        """Normalize the allowed hosts to be a config or None.
-
-        Returns:
-            Optional config.
-        """
-        if self.allowed_hosts and isinstance(self.allowed_hosts, list):
-            self.allowed_hosts = AllowedHostsConfig(allowed_hosts=self.allowed_hosts)
+    experimental_features: frozenset[ExperimentalFeatures] = frozenset()
```

Comment:
Also a breaking change for the reasons stated above

Reply:
Reverted